programmers/음양-더하기.js

```diff
@@ -0,0 +1,31 @@
+// reduce 사용.
+//
+function solution(absolutes, signs) {
+    const reducer = (previousValue, currentValue) => previousValue + currentValue;
+    
+    for (let i = 0; i < absolutes.length; i += 1) {
+        signs[i] === false ? 
+            absolutes.push(-absolutes[i] * 2)
+            :
+            absolutes[i] = absolutes[i];
+    }
+    return absolutes.reduce(reducer);
+}
```

Comment:
```suggestion
function solution(absolutes, signs) {

  return absolutes.reduce((acc, cur, idx) => {
      return signs[idx] ? acc + cur : acc - cur;
  }, 0);
}
```
이 코드 보고 해석해오는게 숙제입니다 ~